### DISCUSSION TITLE
✅ 🚀 Feature Request: Global Section (not Widget)

### DISCUSSION THREAD

**Discussion by webbfox88** [elementor/elementor#21168] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hello,
as I can see this is a huge bug for Elementor (others example DIVI they have it).
You miss the possibility to create a Global Section (yes, the widget can be global but the section is important also).
In this way you can edit a section and automatically Elementor change it in all pages you used a global Section.

The way is similar to Global widget, but it is bigger.
Please solve it as soon as possible.

### Describe the solution you'd like

 you can edit a section and automatically Elementor change it in all pages you used a global Section

### Describe alternatives you've considered

I tried some tricks online but any tricks lose CSS on Mobile Version

**Comment by pumadog** [comment]
Yes, would be nice. Now you have to use a section template with the template widget, a bit more complicated.

**Reply by webbfox88** [reply]
Yes, but when you use this  method you lose CSS setting on mobile for that section.

**Reply by kdub718** [reply]
Yes and there is a major bug in there with containers sizes and responsiveness when you use the template method....

**Comment by luhcm** [comment]
I would love to be able to save a container as a global widget. It takes so much time to update container settings that repeat throughout the website. I can save each widget as global, which does help, but saving the whole container would make the workflow so much smoother!

Big thumbs up for this request.

**Comment by Henzxon** [comment]
Divi does a great job of working with Global section. And you can also disable sync on certain elements within the global section/container so that it won't inherit the global section settings like you can override the button style on specific page but the section still remains global.

**Reply by lensbreak** [reply]
Not a fan of Divi (starts with their not very sustainable business model) but it's a shame they are moving forward much faster than elementor

**Comment by lensbreak** [comment]
+1 waiting for this since very long

**Comment by nicholaszein** [comment]
Hello, there!

Thanks for submitting your suggestion! 🙏  

### ✅ This feature is already available. 

You can use a Section Template, and then use either the [Template widget (Pro)](https://elementor.com/help/template-widget-pro/), or the [Template Shortcode](https://elementor.com/help/how-to-use-shortcodes/#:~:text=the%20Advanced%20Tab.-,Elementor%20Shortcodes,-In%20addition%20to) to add Global Sections to your pages. Once you edit them, you will apply the changes to all pages that include the template. 🙂 

### This also works for Containers in the same fashion. 

All you have to do is to create a Container template or [save a specific Container as a Template](https://elementor.com/help/template-library/#:~:text=template%20and%20save-,Saving%20Sections%20as%20Templates,-Choose%20the%20section) using the [Right-click Context Menu](https://elementor.com/help/right-click/). Then you can use the same method above, and edit your template.

This adds more flexibility to the workflow and allows you to have a centralized hub for your global content. 💪 

I'm closing this request as **`✅ Shipped`**, as this option has been available since Elementor 2.0.

Thanks for contributing! 🙏
